collapsable-cards.js

```diff
@@ -226,58 +189,10 @@ class CollapsableCards extends HTMLElement {
 
 	getStyles() {
 		return `
-      .toggle-button-${this.id} {
-        color: var(--primary-text-color);
-        text-align: left;
-        background: none;
-        border: none;
-        margin: 0;
-        display: flex;
-        justify-content: space-between;
-        align-items: center;
-        width: 100%;
-        border-radius: var(--ha-card-border-radius, 4px);
-        ${this._config.buttonStyle || ''}
-      }
-      .toggle-button-${this.id}:focus {
-        outline: none;
-        background-color: var(--divider-color);
-      }
-
-      .card-list-${this.id} {
-        position: absolute;
-        width: 1px;
-        height: 1px;
-        margin: 0;
-        padding: 0;
-        overflow: hidden;
-        clip: rect(0 0 0 0);
-        clip-path: inset(50%);
-        border: 0;
-        white-space: nowrap;
-      }
-
-      .card-list-${this.id}.is-toggled {
-        position: unset;
-        width: unset;
-        height: unset;
-        margin: unset;
-        padding: unset;
-        overflow: unset;
-        clip: unset;
-        clip-path: unset;
-        border: unset;
-        white-space: unset;
-      }
-
-      .toggle-button__icon-${this.id} {
-        color: var(--paper-item-icon-color, #aaa);
-      }
-
-      .type-custom-collapsable-cards {
-        background: transparent;
-      }
-    `;
+        #root {
+          margin: 0 -8px;
+        }

```

Comment:
For me it breaks the card which contains collapsable-cards. It's better to remove padding from `.container` (`card` variable in the code).

Reply:
I was wrong. `.container` is in shadow dom and it contains element which is `card` in the code. I tried add `this.id` to `#root` and it fixed my problem. But still we don't have nice solution for styling elements.

Reply:
You're right! I don't know how it worked for me before, but I tried a lot of inner levels and maybe in some kind of card it worked. I'll fix it in few hours.

For now I want to keep PR clean and focus to default styles and behavioral.
New options want to add to new PR, like ability to disable outline(saw your screenshot), add styles to header and content, sub-title and more.

Reply:
I think I solved that. What do you think? 
![Snímek obrazovky z 2022-11-17 13-32-13](https://user-images.githubusercontent.com/5030499/202447717-65eb4bc4-443b-48c0-86c3-413bc7a47d6f.png)
![Snímek obrazovky z 2022-11-17 13-32-39](https://user-images.githubusercontent.com/5030499/202447747-2704f8ef-b0b1-43aa-8c62-d937faf9dab4.png)

Reply:
I briefly checked the code. Looks promising, but I'm a bit worried with changing `#root` style. I'll test it in the evening.

Reply:
Firstly I don't think we should provide to users changing styles(don't know any core card have this ability). But it, should be possible if we destroy `getStyles()` method, create a property with same default value as `getStyles()` and add config option to change it. I have prepared another small change that remove `style` attr from `ha-expansion-panel` (removing padding), because this version kills custom styles totally.

All of this work is so much hard in pure JS. All components are in TS using React FW. It would be nice to just rewrite it all, add npm dependecies and build actions. Everything would be easier, but I'm not JS or TS dev so I don't know nothing about building TS source and npm's packages. And don't have time to rewrite it all to TS.

Reply:
I've just tested it and I'm pretty sure that we can remove `getStyles()` because these rules are already there.

Reply:
No they are not. If you using this card in another one, than it can looks like it, but if you remove getStyles(), than all these rules disappear.

Maybe we can remove them and let users use what thay want because these rules are from `vertical-stack` card. So, they can use `vertical-stack` instead of direct cards. I don't know what is better, because without these rules, it works but there's no padding between cards and between first card and header toggler.

Reply:
That's may be the case. I haven't much time yesterday and I tested it quickly in `vertical-stack`.